lib/liquid-fixpoint/src/cstr2smt2.rs

```diff
@@ -670,3 +677,201 @@ pub(crate) fn is_constraint_satisfiable<T: Types>(
     solver.pop(1);
     res
 }
+
+#[derive(Debug)]
+pub enum Z3DecodeError {
+    /// FIXME: (ck) For some reason Z3 dies when doing ast queries on quantifiers (at
+    /// least in testing the formuals we send to it --- it's possible I've done
+    /// something wrong). We don't want quantifiers in our output anyway, so it's fine,
+    /// but we ought to be able to properly deserialize output from Z3.
+    ContainsQuantifier,
+    /// See [`Z3DecodeError::ContainsQuantifier`]. Pretty sure only quantifiers
+    /// introduce Vars.
+    ContainsVar,
+    /// Right now we only expect to see Ints for numerals (can and probably
+    /// should change in the future).
+    UnexpectedSortKindForNumeral(SortKind),
+    UnexpectedAstKind(AstKind),
```

Comment:
This code doesn't compile with `--feature rust-fixpoint` because you are not importing `AstKind`